### PR TITLE
Omit svc.cluster.local suffix for services within the cluster

### PR DESCRIPTION
**Pull request by revit13**
Signed-off-by: Revital Sur <eres@il.ibm.com>

### CODE REVIEW

File: manager/config/prod/deployment_configmap.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ kind: ConfigMap
 metadata:
   name: m4d-config
 data:
-  SECRET_PROVIDER_URL: "http://secret-provider.m4d-system.svc.cluster.local:5555/get-secret"
+  SECRET_PROVIDER_URL: "http://secret-provider:5555/get-secret"
```
Comment (roee88) [inline manager/config/prod/deployment_configmap.yaml:9]: This won't work, it is passed to modules that run in different namespace

**Review by roee88 — CHANGES_REQUESTED** [review]
Added comment in secret provider.
Also, have you tested it on kind/OpenShift?

File: manager/config/prod/deployment_configmap.yaml (hunk 1 above)
Comment (revit13) [reply manager/config/prod/deployment_configmap.yaml:9]: I tested it on Kind. The suffixes  in /etc/resolve are: m4d-system.svc.cluster.local svc.cluster.local cluster.local
That's why I omitted the m4d-system

Comment (revit13) [reply manager/config/prod/deployment_configmap.yaml:9]: I validated and the arrow module did not fail... maybe the address is cached somehow? is there a way to look at the logs? thanks

Comment (revit13) [reply manager/config/prod/deployment_configmap.yaml]: I've added the m4d-system to secret-provider. Thanks

**Review by roee88 — APPROVED** [review]

**Pull request merged**